miasm2/ir/ssa.py

```diff
@@ -322,3 +322,20 @@ def reassemble_expr(self, e):
                 if id_rhs in self.expressions:
                     todo.add(id_rhs)
         return e
+
+
+class SSAPath(SSABlock):
```

Comment:
I'm not sure to correctly understand the purpose of `SSAPath`. Is it designed to always work on a path? If it is the case, is a path a list of SSABlock ?

In this last case, `SSAPath` should not inherits from `SSABlock`, but from `UserList` or `list` to get methods such as `append`, iteration, comparison, ... Then it can be initialized with a list of `SSABlock`, and directly works on it (so `transform` method will apply on its internal path).

Am I correct?